src/Slides.js

```diff
@@ -0,0 +1,28 @@
+import { URL } from 'url';
+
+const getTrimmedPathName = pathname => pathname.replace(/\/+$/, '');
+
+export const shouldTransform = url => {
+  const { host, pathname } = new URL(url);
+
+  return (
+    ['slides.com', 'www.slides.com', 'team.slides.com'].includes(host) &&
+    getTrimmedPathName(pathname)
+      .split('/')
+      .filter(Boolean).length === 2
```

Comment:
actual filter is needed because getTrimmedPathName trims trailing slash not all the slash

Reply:
And also we are checking `.length === 2` so if we remove filter and url is malformed or if pathname array have some extra values in it then it will stop working.

Reply:
That's the only thing we need no?

Because `'username/deck-name/////'` will become `'username/deck-name'`, so if you split you'll have `['username', 'deck-name']`.

Reply:
in pathname we also have / in front of a path so i guess also have to remove the 1st slash.

Reply:
Yeah indeed, the first slash needs to be removed by `getTrimmedPathName` too then for this to work.

Reply:
got it